### PR TITLE
fix: make dashboard link to open in new tab for saved charts

### PR DESCRIPTION
**Pull request by muhsinkamil**
Closes: #6188 

### Description:

- Make link to open in new tab

### CODE REVIEW

**Comment by rephus** [comment]
I'll duplicate this branch on Lightdash to test this on render, @magnew could you review and test the code ?

File: packages/frontend/src/components/common/ResourceInfoPopup/DashboardList.tsx [hunk 1]
```diff
@@ -25,8 +24,8 @@ export const DashboardList: FC<Props> = ({ resourceItemId, projectUuid }) => {
                     {relatedDashboards.map(({ uuid, name }) => (
                         <List.Item key={uuid}>
                             <Anchor
-                                component={Link}
-                                to={`/projects/${projectUuid}/dashboards/${uuid}/view`}
+                                href={`${window.location.origin}/projects/${projectUuid}/dashboards/${uuid}/view/`}
```
Comment (IrakliJani) [inline packages/frontend/src/components/common/ResourceInfoPopup/DashboardList.tsx:27]: changing from Link (react-router-dom) removes SPA navigation functionality (refreshes the page) my understanding is that `<Anchor component={Link}` supports opening to a new tab. maybe this bug has something to do something with `e.preventDefault()`?

Comment (muhsinkamil) [reply packages/frontend/src/components/common/ResourceInfoPopup/DashboardList.tsx:27]: Yes, it boots up / refreshes the app again for a new tab. Its not above `e.preventDefault()`, I tested by removing just that and it doesn't open in new tab.,
And as far as I know, `Link` from react-router is only for internal routing, here we are trying to open in a new tab, hence going out. 
Correct me if I'm wrong please

Comment (IrakliJani) [reply packages/frontend/src/components/common/ResourceInfoPopup/DashboardList.tsx:27]: ok! current changes look good then!

**Review by IrakliJani — APPROVED** [review]

File: packages/frontend/src/components/common/ResourceInfoPopup/DashboardList.tsx (hunk 1 above)
Comment (yasmin-e) [reply packages/frontend/src/components/common/ResourceInfoPopup/DashboardList.tsx:27]: Thank you for your contribution @muhsinkamil! ⚡ 🙌🏼

**Pull request merged**